app/lib/submission_builder/manifest.rb

```diff
@@ -0,0 +1,28 @@
+module SubmissionBuilder
+  class Manifest
+    include Buildable
+
+    SCHEMA_FILE = File.join(Rails.root, "vendor", "irs", "unpacked", "2020v5.1", "Common", "efileAttachments.xsd")
+
+    def initialize(submission)
+      @submission = submission
+      @intake = submission.intake
+    end
+
+    def build
+      document = Nokogiri::XML::Builder.new do |xml|
+        xml['efil'].IRSSubmissionManifest("xmlns:efil" => "http://www.irs.gov/efile", "xmlns" => "http://www.irs.gov/efile") {
+          xml.SubmissionId @submission.irs_submission_id
+          xml.EFIN EnvironmentCredentials.dig(:irs, :efin)
+          xml.GovernmentCd "IRS"
+          xml.FederalSubmissionTypeCd "1040"
+          xml.TIN @intake.primary_ssn
+        }
+      end
+
+      xsd = Nokogiri::XML::Schema(File.open(SCHEMA_FILE))
+      xml = Nokogiri::XML(document.to_xml)
```

Comment:
In the long run, this will be the same across all `Buildable`s. Opportunity for refactor?